.travis.yml

```diff
@@ -13,14 +13,6 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
-      env:
-        - DEPS=latest
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
```

Comment:
We cannot drop versions from testing that are explicitly supported via the package constraints. You either need to bump the PHP constraint to `^7.2`, or keep the above.

Reply:
PHPUnit 8.x doesn't support PHP 7.1 and PHPUnit 7.x doesn't support 8.x. What do you recommend in this case?